src/public/map.js

```diff
@@ -95,17 +66,35 @@ map.on('draw:edited', function (e) {
     // loops over each edited layer
     // do whatever you want, most likely save back to db
   });
-  doPost("/search.sjs", "name", displayGeoJSON, drawnShapes);
+  doPost("/search.sjs", "name",displayGeoJSON, drawnShapes, false);
 });
 
 map.on('draw:deleted', function (e) {
   // Update db to save latest changes.
   drawnShapes.removeLayer(e.layer);
 });
 
-// ****** Copied from Jen and Jake's geoapp ********
-function doPost(url, str, success, drawnLayer) {
-  //clearResults();
+
+function populateMenus(response) {
+  displayFeatures(response.features.facets);
+  displayIndustries(response.industries.facets);
+}
+
+function displayFeatures(features) {
+  for (var obj in features.Features) {
```

Comment:
I think it would make sense to add `$("#collapse2 ul").empty()` before the loop, right?